### PR TITLE
remove upgrade information from ls and inspect command

### LINKED ISSUE #3277: Upgrade status shows in vic-machine ls and inspect output
The upgrade option has been hidden in version 0.8, but the upgrade status still appears in the output of `vic-machine ls` and `vic-machine inspect` if you have older VCHs. This is misleading, as no upgrade to 0.8 is available.

Seen with version `v0.8.0-rc1-7157-260dbce`.

### PR DESCRIPTION
**Pull request by emlin**
fixes #3277

### CODE REVIEW

**Comment by mdubya66** [comment]
lgtm

File: cmd/vic-machine/inspect/inspect.go [hunk 1]
```diff
@@ -136,9 +136,9 @@ func (i *Inspect) Run(cli *cli.Context) error {
 	log.Info("")
 	log.Infof("Installer version: %s", installerVer.ShortVersion())
 	log.Infof("VCH version: %s", vchConfig.Version.ShortVersion())
-	log.Info("")
-	log.Info("VCH upgrade status:")
-	i.upgradeStatusMessage(ctx, vch, installerVer, vchConfig.Version)
+	//	log.Info("")
+	//	log.Info("VCH upgrade status:")
+	//	i.upgradeStatusMessage(ctx, vch, installerVer, vchConfig.Version)
```
Comment (cgtexmex) [inline cmd/vic-machine/inspect/inspect.go]: Is this just commented out because we will reintroduce it later?

**Comment by chengwang86** [comment]
lgtm

BTW, I don't find anything related to "upgrade" when I do "docker inspect" even without this fix. Did I miss anything?

**Comment by jzt** [comment]
Like @cgtexmex, I'm curious about the commented code, but LGTM

File: cmd/vic-machine/inspect/inspect.go (hunk 1 above)
Comment (emlin) [inline cmd/vic-machine/inspect/inspect.go]: right

**Comment by emlin** [comment]
yeah, upgrade is just removed temporarily, so I don't want to remove any code lines if I can.

**Comment by cgtexmex** [comment]
lgtm

**Comment by emlin** [comment]
@chengwang86 this is for vic-machine inspect command

**Comment by vburenin** [comment]
LGTM

**Pull request merged**